tests/src/test/scala/org/apache/openwhisk/core/containerpool/mesos/test/MesosContainerFactoryTest.scala

```diff
@@ -112,8 +112,8 @@ class MesosContainerFactoryTest
       logging,
       Map("--arg1" -> Set("v1", "v2")),
       containerArgsConfig,
-      mesosConfig,
-      (system, mesosConfig) => testActor)
+      mesosConfig = mesosConfig,
+      clientFactory = (system, mesosConfig) => testActor)
```

Comment:
Is the diff in this file necessary?

Reply:
Yes, as I'm adding the new `RuntimesRegistryConfig` to Mesos as well.